test/unit/app/models/test_models.py

```diff
@@ -330,23 +330,34 @@ def test_new_credit_transfer_check_sender_transfer_limits(create_credit_transfer
         if 'Sempo Level 3: P' in limit.name
     ]
 
-    # Check GE LIMITS for Liquid Token (withdrawal)
+    # Check additional GE LIMITS for Liquid Token (withdrawal)
     create_credit_transfer.token.token_type = token.TokenType.LIQUID
     create_credit_transfer.transfer_type = TransferTypeEnum.WITHDRAWAL
     create_credit_transfer.sender_transfer_account.balance = 10000
     assert create_credit_transfer.check_sender_transfer_limits() == [
         limit for limit in LIMITS
-        if 'GE Liquid Token - Standard User' == limit.name
+        if 'GE Liquid Token - Standard User' == limit.name or 'Sempo Level 3: WD' in limit.name
```

Comment:
TODOs:
- refactor test_models.py into file for each model
- refactor test_new_credit_transfer_check_sender_transfer_limits into sperate tests